templates/donor-donation-success.html.twig

```diff
@@ -1,5 +1,40 @@
 {% extends  '_base.html.twig' %}
 
 {% block content %}
-  <p>Thanks for donating, {{ donorFirstName }}!</p>
+  <p>Dear {{ donorFirstName }} {{ donorLastName }},</p>
+  <p>Thank you for supporting {{ charityName }} with your generous gift through the Big Give.<p>
+  <p>
+    Here is your donation information:
+      <ul>
+        <li>Total for {{ charityName }}: <strong>£{{ totalCharityValueAmount }}</strong></li>
+        <li>Donation: <strong>£{{ donationAmount}}</strong></li>
+        <li>Gift Aid: <strong>£{{ giftAidAmountClaimed }}</strong></li>
+        {% if matchedAmount > 0 %}
+            <li>Match Funds: <strong>£{{ matchedAmount }}</strong></li>
+        {% endif %}
+      </ul>
+
+  <p>
+    Donation reference: <strong>{{ transactionId }}</strong><br>
+    Reference on credit/debit card statement: <span>Charity</span>Checkout<span>.co</span>.uk
+  </p>
+
+  {% if tipAmount > 0 %}
+    <p>Thank you for leaving a tip of <strong>£{{ tipAmount }}</strong> toward the Big Give.</p>
+  {% endif %}
+
+  <p>Total amount you have been charged is <strong>£{{ totalChargedAmount }}</strong>.</p>
+
+  {# {% if comms %}
```

Comment:
How's this going to work?

Reply:
Not quite sure tbh, I found it in the pardot email template and wondered if it's worth keeping

Reply:
If we don't know why it's there it should not be going into new production code @tbg-jasonfung!